src/utils/utils.ts

```diff
@@ -35,3 +37,17 @@ export const getDefaultConfig = (webview: Webview, extensionUri: Uri) => {
 };
 
 export const getPositionFragment = (pos: Position) => `L${pos.line + 1}C${pos.character}`;
+
+export const getFilesFromPythonPackages = (file: string, packages: string[]) => {
+  const config = workspace.getConfiguration("python");
+  const pythonPath = config.get("pythonPath", "python");
+  return new Promise<Record<string, string>>((resolve, reject) => {
+    exec(`"${pythonPath}" "${join(__dirname, "python", "find_file_in_package.py")}" "${file}" "${packages.join('" "')}"`, (error, stdout, stderr) => {
```

Comment:
We will see whether __dirname could be used after deploying. I guess it should work fine

Reply:
Yes it will as we are in a node.js process

Reply:
Copy/paste from your code :-)